src/core/mavsdk.h

```diff
@@ -182,6 +192,11 @@ class Mavsdk {
         /** @brief Usage type of this configuration, used for automatic ID set */
         UsageType get_usage_type() const;
 
+        /**
+         * @brief Set the usage type of this configuration.
```

Comment:
Maybe add a comment here:

> Note that this will set `system_id` and `component_id`.

Reply:
You mean a comment on set_usage_type()? That comment would not be accurate, there are separate setter methods for system_id and component_id. You may be confusing this function with MavsdkImpl::set_configuration()?

Reply:
Oh, I might be.